src/appleseed.studio/python/pythoninterpreter.h

```diff
@@ -46,10 +48,14 @@ class PythonInterpreter
     void execute_command(const char* command);
 
     void redirect_output(OutputRedirector redirector);
+    void set_mainwindow(MainWindow* mainWindow);
```

Comment:
I'd suggest reordering methods of this class to follow a more logical progression:

    instance

    set_mainwindow
    get_mainwindow

    redirect_output

    execute_command